store.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Karma Store</title>
+    <link rel="stylesheet" href="store.css" />
+    <link rel="additional page" href="index.html" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="karma logo" id="karma-logo" />
+      <nav id="nav-bar">
+        <a href="index.html" id="karma-meet">Meet Karma</a>
+        <a href="works">How it Works</a>
+        <a href="store">Store</a>
+        <a href="blog">Blog</a>
+        <a href="help">Help</a>
+        <a href="login">Login</a>
```

Comment:
semantically: this navigation should be an unordered list of list items that contain anchors
this is helpful for accessibility
```
<ul>
  <li>
    <a>
```